stats

```diff
@@ -9,7 +10,23 @@
  * file that was distributed with this source code.
  */
 
-require 'vendor/autoload.php';
+$loaded = false;
```

Comment:
can we wrap this whole thing in a closure to avoid defining this global variable please?